plugins/modules/packaging/os/pacman.py

```diff
@@ -43,10 +43,17 @@
 
     force:
         description:
-            - When removing package, force remove package, without any checks.
-              Same as `extra_args="--nodeps --nodeps"`.
+            - When removing packages, forcefully remove them, without any checks.
+              Same as C(extra_args="--nodeps --nodeps").
               When update_cache, force redownload repo databases.
```

Comment:
```suggestion
            - When combined with I(update_cache), force redownload of repo databases.
```

Reply:
Or `force redownloading repo databases`?

Reply:
or we can use `pacman`'s wording: `force a refresh of all package databases`